src/next-transpile-modules.js

```diff
@@ -120,12 +120,15 @@ const withTmInitializer = (modules = [], options = {}) => {
 
     // Generate Webpack condition for the passed modules
     // https://webpack.js.org/configuration/module/#ruleinclude
-    const match = (path) =>
-      modulesPaths.some((modulePath) => {
-        const transpiled = path.includes(modulePath);
+    const match = (path) => {
+      const lastEntry = path.split('/node_modules/').slice(-1)[0];
```

Comment:
This will be an issue on Windows which uses anti-slashes, can you handle this case as well?